app/views/notifications/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<ol>
+  <% @notifications.each do |notification|%>
+    <% if notification.notification_type==="likes"%>
+      <div class="notification">
```

Comment:
Also Delcare a map in Notification type and uses that.
e.g 
```
  NOTIFICATION_TYPES = {
LIKES: "likes"
}
```
And use it like
```
Notification.NOTIFICATION_TYPES[:LIKES]
```

Reply:
Done.
I was accessing the variable without any getter method.